TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -235,7 +235,89 @@ const BenchmarkMetricRegex = {
                 funcName: geomean,
             }
         }
-    }
+    },
+    "quarkus-pingperf_baremetal_throughput": {
+        outerRegex:/Running \d*? measures([\s\S\n]*)/, 
+        //execlude warmup runs from getting parsed
+        metrics: {
+            "Throughput": {
+                //Example: Requests/sec: 100522.45/
+                regex:/Requests\/sec: (\d*\.?\d*)/,
+                higherbetter: true,
+                units: "Requests/sec"
+            },
+        }
+    },    
+    "quarkus-startup": {
+        outerRegex: /Quarkus Test:  measure \d*? ([\s\S\n]*)/,
+        //exclude warmup runs from getting parsed
+        metrics: {
+            "Footprint": {
+                //Example: Footprint for  measure 0 10129332
+                regex:/Footprint for  measure \d*? (\d*\.?\d*)/,
+                higherbetter: false,
+                units: "kb",
+                },
+            "Startup time": {
+                //Example: Startup time for measure 0 : 53232
+                regex:/Startup time for measure \d*? : (\d*\.?\d*)/,
+                higherbetter: false,
+                units: "ms",
+            }
+        }
+    },
+    "quarkusRestCrudDemo": {
+        metrics: {
+            "Throughput": {
+                //Example: Measure 0 ~ Requests/sec: 15145.27 -- in multi user, we test multiple scenarios of thread & connection in 1 run - using geomean as our metric.
+                regex:/Measure[\s\S\n]*?Requests\/sec: (\d*\.?\d*)/,
+                higherbetter: true,
+                units: "Requests/sec",
+                funcName: geomean,
+            },
+        }
+    },
+    "quarkus-getting_started": {
```

Comment:
That should be `getting-started`.

Reply:
Having a short name will cause matching issues in the future as we add more and more data. Unless this needs to be shared with other benchmark tests, I would prefer to keep the full name.

Reply:
Sounds good! Since the folder name is `getting-started`, maybe we can name it `getting-started-<extraInfoToMakeItBigger>` to stick to the convention we decided (`<folderName>-<extraInfo>`). We can call it `getting-started-default` to make it a longer name.

Reply:
The naming convention is `<benchmark>-<extra>`. In the regular case, <folderName> should match with the <benchmark>. In this case, we have two levels: `quarkus` folder and `getting-started` folder.
I think the name should be `quarkus-getting-started`. `getting-started` or `getting-started-default` is too general and does not contain the benchmark name.

Reply:
Sounds good! In that case, we'll prepend `quarkus-` to all tests under `quarkus/<BenchmarkFolderName>` to be consistent if you prefer that.